superbench/benchmarks/micro_benchmarks/gpu_copy_performance/gpu_copy.cu

```diff
@@ -810,13 +808,16 @@ int main(int argc, char **argv) {
                     args_list.push_back(args);
                 }
             }
+            if (args.numa_id != 0) {
+                continue;
+            }
             // Device-to-device benchmark
             if (opts.dtod_enabled) {
                 // Scan all peers
                 for (int k = 0; k < gpu_count; k++) {
-                    // Skip second half for bidirectional test
-                    if (opts.bidirectional_enabled && k > j) {
-                        break;
+                    // src_dev_id always < dst_dev_id for bidirectional test
+                    if (opts.bidirectional_enabled && j >= k) {
```

Comment:
Why change this logic?

Reply:
Currently Bi P2P results are displayed as gpu1_to_gpu0…, this changes it to gpu0_to_gpu1…, looks better. Also it skips self to self bi test.